src/nls/sv/strings.js

```diff
@@ -370,11 +373,14 @@ define({
     "VIEW_MENU"                           : "Visa",
     "CMD_HIDE_SIDEBAR"                    : "Dölj sidomeny",
     "CMD_SHOW_SIDEBAR"                    : "Visa sidomeny",
+    "CMD_TOGGLE_SIDEBAR"                  : "Växla på/av sidomeny",
+    "CMD_TOGGLE_PANELS"                   : "Växla på/av paneler",
+    "CMD_TOGGLE_PURE_CODE"                : "Inga distraktioner",
     "CMD_INCREASE_FONT_SIZE"              : "Öka textstorlek",
     "CMD_DECREASE_FONT_SIZE"              : "Minska teckenstorlek",
     "CMD_RESTORE_FONT_SIZE"               : "Återställ teckenstorlek",
-    "CMD_SCROLL_LINE_UP"                  : "Skrolla rad uppåt",
-    "CMD_SCROLL_LINE_DOWN"                : "Skrolla rad nedåt",
+    "CMD_SCROLL_LINE_UP"                  : "Rulla rad uppåt",
+    "CMD_SCROLL_LINE_DOWN"                : "Rulla rad nedåt",
```

Comment:
Based on [recommendation by Datatermgruppen](http://www.datatermgruppen.se/ordlista.html#b95) and my own preference. But I agree that "skrolla" is perfectly understandable.